sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -529,7 +529,7 @@ object FunctionRegistry {
     expression[BitwiseNot]("~"),
     expression[BitwiseOr]("|"),
     expression[BitwiseXor]("^"),
-    expression[BitwiseCount]("bit_count"),
+    expression[BitwiseCount]("bit_count", true),
```

Comment:
This seems already correct? https://github.com/apache/spark/blob/a5efbb284e29b1d879490a4ee2c9fa08acec42b0/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/bitwiseExpressions.scala#L188

Reply:
I don't think it's elegant enough. Because `Expression` gives the following definitions.
```
  def sql: String = {
    val childrenSQL = children.map(_.sql).mkString(", ")
    s"$prettyName($childrenSQL)"
  }
```
By this definition, I judge legitimacy by comparing `prettyName` with alias.

Reply:
Okay, sticking with `prettyName` is okay. Can we change `prettyName` in `BitwiseCount` to `bit_count` then? Seems it's right to change.

Reply:
OK. I will open another PR to manually handle the aliases.